### PR TITLE
feat: add documentation for slider and hamburger/drawer

### PR DESCRIPTION
**Pull request by shaik-saad**
Screen shots:

Screenshot 2022-02-23 084210
Screenshot 2022-02-23 084236

### CODE REVIEW

**Review by BISINIGISHIVANI — APPROVED** [review]
code looks fine

File: documentation/components/slider.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    <!-- Footer ends here-->
+
+    <!-- Highlight.js Script Tags -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+    <script>hljs.highlightAll();</script>
+
+    <!-- Site script Tag-->
+    <script src="../doc-site.js"></script>
+</body>
+</html>
```
Comment (VishalPatil18) [inline documentation/components/slider.html:212]: You could add an empty line at the end of the file to avoid such warnings by Github.

**Review by VishalPatil18 — APPROVED** [review]
- The sidebar is getting overflowed on smaller screen sizes.
Screenshot 2022-02-23 at 09 06 19

> Apart from this the rest of the code looks quite good. Great work @shaik-saad 👍

File: documentation/components/slider.html (hunk 1 above)
Comment (shaik-saad) [reply documentation/components/slider.html:212]: sure, will do it

**Pull request merged**